sdk/search/open-api/getSpecsAndGenerate.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I'm wondering if this should be generated using our existing code generation infrastructure.

Reply:
there's a PR in the api spec repo with the autrest config (Happy new year). This script was used during the development and can be removed, pending PR to confirm

Reply:
Happy New Year :)